core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -2162,6 +2162,48 @@ private SqlDialect nonOrdinalDialect() {
         .ok(prestoExpected);
   }
 
+  /**
+   * Test case for the base case of
+   * <a href="https://issues.apache.org/jira/browse/CALCITE-6355">[CALCITE-6355]
```

Comment:
So far we have required the JIRA case title, the PR case title, and the comments in the code to use the exact same text.
This makes it easy to understand how these are related.
Can you please follow these guidelines?
There are lots of examples.

Reply:
Sorry, missed this. I've adjusted the title -- thank you. Do I have to reword commits? Or the title of the PR will be used when squashed?

Reply:
Eventually after the PR is accepted you will have to squash all the commits to just one, and that one will have to have the exact same message as the JIRA title, and this PR title. Moreover, the JavaDoc for the test cases related to the fix also have to have a relatively rigid format, providing the link to the jira case and the full text in the jira title.